docs/nodejs.md

```diff
@@ -0,0 +1,29 @@
+# Node.js
+
+Altis supports running Node.js applications alongside WordPress, utilizing WordPress as a headless API.
+
+## Enabling Node.js in Local Server
+
+Node.js can be enabled in Local Server by adding `extra.altis.modules.local-server.nodejs` in the project's `composer.json`
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"local-server": {
+					"nodejs": {
+            "path": "../altis-nodejs-skeleton",
+            "version": "21.1"
```

Comment:
Can we instead load the version from `${path}/package.json`? That's how we ask clients to specify their node version in Altis Cloud.

Reply:
Implemented